syntax = "proto3";

package lms_proto;

import "google/protobuf/empty.proto";

service HomeworkController {
  rpc Validation(CourseRequest) returns (CourseReply) {}
  rpc HomeworkID(FileRequest) returns (FileReply){}
}

message CourseRequest {
  string user_id = 1;
  string course_id = 2;
}

message CourseReply {
  bool valid_user = 1;
  bool valid_course = 2;
}

message FileReply {
  string hw_url = 1;
}

message FileRequest {
  string hw_id = 1;
}
